### DISCUSSION TITLE
Multi LDAP change - help needed

### DISCUSSION THREAD

**Discussion by scrensen** [ltb-project/self-service-password#643] [Q&A]
Hi all, I'm trying to get the multi_ldap_change.php to work, but I need it to work with different credentials.

I'm not a developer, just an enthousiast component owner that likes to make this tool available to our customers.

What I was hoping what would work is to add this to my config.inc.local.php:

$posthook = "php /usr/share/self-service-password/scripts/multi_ldap_change.php";
$display_posthook_error = true;
$secondaries_ldap[0]['ldap_url'] = 'ldap://ldp01.world.com';
$secondaries_ldap[0]['ldap_binddn'] = 'cn=blaaccount,ou=AdminAccounts,dc=world,dc=com';
$secondaries_ldap[0]['ldap_bindpw'] = 'blabla123';
$posthook_password_encodebase64 = true;

But it seems it doesn't work, as I don't even see a connection attempt in the logs

I did manage to get it to work by adding 2 variables to multi_ldap_change.php:

$ldap_binddn2 = 'cn=blaaccount,ou=AdminAccounts,dc=world,dc=com';
$ldap_bindpw2 = 'blabla123';

And then modify some code below to use them. But I think this is rather ugly and was hoping to do it via the config file.

Anyone any suggestions perhaps?

**Comment by coudot** [comment]
There should be a log in /tmp/multi_ldap_change.log, could you see if there is any error in it?
